### PR TITLE
Modified for LEONARDO

### PR DESCRIPTION
**Pull request by asficas**
Leonardo uses a quartz so it the timing is more accurate.
The pins used are 15, 14 and 16 for the rx5808 and 3 for the rssi

User has to uncomment  //#define LEONARDO at line 34

### CODE REVIEW

File: src/delta5node/delta5node.ino [hunk 1]
```diff
@@ -31,9 +31,16 @@
...
 #define i2cSlaveAddress 8
 
-const int slaveSelectPin = 10; // Setup data pins for rx5808 comms
-const int spiDataPin = 11;
-const int spiClockPin = 13;
+//#define LEONARDO
+#ifdef LEONARDO
+	const int slaveSelectPin = 15; // Setup data pins for rx5808 comms
+	const int spiDataPin = 14;
+	const int spiClockPin = 16;
```
Comment (punkkills) [inline src/delta5node/delta5node.ino]: I would leave the tabbing as it was

File: src/delta5node/delta5node.ino [hunk 2]
```diff
@@ -31,9 +31,16 @@
 // Node 5 = 16, Node 6 = 18, Node 7 = 20, Node 8 = 22
 #define i2cSlaveAddress 8
 
-const int slaveSelectPin = 10; // Setup data pins for rx5808 comms
-const int spiDataPin = 11;
-const int spiClockPin = 13;
+//#define LEONARDO
```
Comment (punkkills) [inline src/delta5node/delta5node.ino]: maybe a comment here saying "uncomment the line for use with..."

File: src/delta5node/delta5node.ino [hunk 3]
```diff
@@ -254,7 +261,11 @@ void setRxModule(int frequency) {
 
 // Read the RSSI value for the current channel
 int rssiRead() {
-	return analogRead(0);
+	#ifdef LEONARDO
```
Comment (punkkills) [inline src/delta5node/delta5node.ino]: instead of the define here, can you add a variable for the pin with the other variables?

**Comment by asficas** [comment]
I have made requested changes.

File: src/delta5node/delta5node.ino (hunk 2 above)
Comment (asficas) [reply src/delta5node/delta5node.ino]: ok

File: src/delta5node/delta5node.ino (hunk 1 above)
Comment (asficas) [reply src/delta5node/delta5node.ino]: ok

File: src/delta5node/delta5node.ino (hunk 3 above)
Comment (asficas) [reply src/delta5node/delta5node.ino]: ok

**Comment by asficas** [comment]
Are there any issues remaining?

**Pull request closed without merging**